prdoc/pr_9938.prdoc

```diff
@@ -0,0 +1,17 @@
+title: 'Snowbridge - Adds Fulu hardfork'
+doc:
+- audience: Runtime Dev
+  description: Set fork version for Fulu hardfork on Sepolia.
+crates:
+- name: snowbridge-pallet-ethereum-client
+  bump: patch
+- name: snowbridge-pallet-inbound-queue
+  bump: patch
+- name: snowbridge-beacon-primitives
+  bump: major
+- name: snowbridge-beacon-primitives
+  bump: patch
```

Comment:
patch or major?

```suggestion
- name: snowbridge-beacon-primitives
  bump: major
- name: snowbridge-beacon-primitives
  bump: patch
```

Reply:
`major`, addressed in https://github.com/paritytech/polkadot-sdk/pull/9938/commits/2b80d3782e2dc1d98e24cf7396d71e39ce2104df.

Reply:
Thanks for the tip, changed to `patch`